2017-01-13 15:33:31  [ main:0 ] - [ DEBUG ]  正在匹配：--------------------3451---------------------
2017-01-13 15:33:31  [ main:5 ] - [ ERROR ]  正在匹配：--------------------3451---------------------
2017-01-13 15:33:31  [ main:23 ] - [ ERROR ]  这条轨迹匹配出现问题：3451
2017-01-13 15:33:31  [ main:23 ] - [ DEBUG ]  匹配结束：------------------3451-----------------------
2017-01-13 15:33:31  [ main:24 ] - [ ERROR ]  匹配结束：------------------3451-----------------------
2017-01-13 15:57:47  [ main:0 ] - [ DEBUG ]  正在匹配：--------------------3451---------------------
2017-01-13 15:57:47  [ main:3 ] - [ ERROR ]  正在匹配：--------------------3451---------------------
2017-01-13 15:57:54  [ main:7282 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:57:54  [ main:7658 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:57:54  [ main:7762 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:57:55  [ main:8872 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:57:58  [ main:11221 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:00  [ main:13813 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:03  [ main:16404 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:03  [ main:16464 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:04  [ main:17022 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:06  [ main:19392 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:07  [ main:20322 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:12  [ main:25103 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:20  [ main:33256 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:20  [ main:33591 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:23  [ main:36229 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:24  [ main:37297 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:58:24  [ main:37394 ] - [ ERROR ]  [ndm.NetworkAnalyse.Match.SolveTimeExceptionPoint(Match.java:706), ndm.NetworkAnalyse.Match.MapMatch(Match.java:359), ndm.NetworkAnalyse.MatchProcess.mapmatching(MatchProcess.java:83), ndm.NetworkAnalyse.MatchProcess.main(MatchProcess.java:32)]
2017-01-13 15:58:24  [ main:37395 ] - [ DEBUG ]  匹配结束：------------------3451-----------------------
2017-01-13 15:58:24  [ main:37395 ] - [ ERROR ]  匹配结束：------------------3451-----------------------
2017-01-13 15:59:21  [ main:0 ] - [ DEBUG ]  正在匹配：--------------------3451---------------------
2017-01-13 15:59:21  [ main:2 ] - [ ERROR ]  正在匹配：--------------------3451---------------------
2017-01-13 15:59:28  [ main:6588 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:28  [ main:6903 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:28  [ main:6994 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:29  [ main:7875 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:31  [ main:10240 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:34  [ main:12613 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:36  [ main:14654 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:36  [ main:14715 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:36  [ main:15107 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:38  [ main:17109 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:39  [ main:17815 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:42  [ main:21035 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:47  [ main:26279 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:48  [ main:26558 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:49  [ main:28442 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:50  [ main:29243 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 15:59:50  [ main:29284 ] - [ ERROR ]  [oracle.jdbc.driver.OraclePreparedStatement.executeBatch(OraclePreparedStatement.java:11190), oracle.jdbc.driver.OracleStatementWrapper.executeBatch(OracleStatementWrapper.java:244), ndm.NetworkAnalyse.Match.SolveTimeExceptionPoint(Match.java:589), ndm.NetworkAnalyse.Match.MapMatch(Match.java:359), ndm.NetworkAnalyse.MatchProcess.mapmatching(MatchProcess.java:83), ndm.NetworkAnalyse.MatchProcess.main(MatchProcess.java:32)]
2017-01-13 15:59:50  [ main:29284 ] - [ DEBUG ]  匹配结束：------------------3451-----------------------
2017-01-13 15:59:50  [ main:29284 ] - [ ERROR ]  匹配结束：------------------3451-----------------------
2017-01-13 16:02:11  [ main:0 ] - [ DEBUG ]  正在匹配：--------------------3451---------------------
2017-01-13 16:02:11  [ main:2 ] - [ ERROR ]  正在匹配：--------------------3451---------------------
2017-01-13 16:02:18  [ main:6624 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:18  [ main:6944 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:18  [ main:7036 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:19  [ main:7890 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:21  [ main:10229 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:24  [ main:12609 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:26  [ main:14702 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:26  [ main:14768 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:26  [ main:15200 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:28  [ main:17327 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:29  [ main:18055 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:33  [ main:21463 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:38  [ main:27076 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:39  [ main:27356 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:41  [ main:29357 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:41  [ main:30221 ] - [ DEBUG ]  计算距离出现异常！135行
2017-01-13 16:02:41  [ main:30271 ] - [ ERROR ]  [oracle.jdbc.driver.OraclePreparedStatement.executeBatch(OraclePreparedStatement.java:11190), oracle.jdbc.driver.OracleStatementWrapper.executeBatch(OracleStatementWrapper.java:244), ndm.NetworkAnalyse.Match.SolveTimeExceptionPoint(Match.java:589), ndm.NetworkAnalyse.Match.MapMatch(Match.java:359), ndm.NetworkAnalyse.MatchProcess.mapmatching(MatchProcess.java:83), ndm.NetworkAnalyse.MatchProcess.main(MatchProcess.java:32)]
2017-01-13 16:02:41  [ main:30272 ] - [ DEBUG ]  匹配结束：------------------3451-----------------------
2017-01-13 16:02:41  [ main:30272 ] - [ ERROR ]  匹配结束：------------------3451-----------------------
2017-01-13 16:07:08  [ main:0 ] - [ DEBUG ]  正在匹配：--------------------3451---------------------
2017-01-13 16:07:08  [ main:2 ] - [ ERROR ]  正在匹配：--------------------3451---------------------
